### DISCUSSION TITLE
Brainstorm da Feature Cadastro e Manutenção

### DISCUSSION THREAD

**Discussion by crvzybird** [crvzybird/ess-cardapio#16] [Ideas]
## :pencil2: Bem vindo(a)!
  Essa **_Discussion_**  servirá como um espaço para realizar **_Brainstorms_** da **_Feature_**  **Cadastro e Manutenção**  e qualquer etapa de desenvolvimento dela que esteja na sua faze de conceituação e ideação.Com o intuito de podermos nos manter conectados mesmo não sendo possível estar em reuniões virtuais/presenciais é esperado que comentários sejam feitos para debater possíveis abordagens, implementações e até sugestão de refatorações de implementações que possam contribuir no desenvolvimento dessa *_Feature_**

**Comment by crvzybird** [comment]
@costa402 Como estaríamos  tratando a questão das categorias? A feature é cadastro e edição de itens no cardápio, mas Categorias seria considerado como parte do item ou outro fator a parte?

**Reply by costa402** [reply]
Creio que seria interessante por questões de UX que o item já aparecesse no cardápio assim que fosse cadastrado e pra isso era interessante que o item já nascesse com uma categoria associada

**Comment by costa402** [comment]
Pelo menos na minha visão eu entendo que seria melhor dessa forma do que o usuario ter que linkar manualmente itens e categorias depois
